VideoLocker/src/main/java/org/edx/mobile/services/EdxCookieManager.java

```diff
@@ -98,12 +96,6 @@ public void onSuccess(List<HttpCookie> result) {
                             clearWebWiewCookie(context);
                             PrefManager pref = new PrefManager(MainApplication.instance(), PrefManager.Pref.LOGIN);
                             pref.put(PrefManager.Key.AUTH_ASSESSMENT_SESSION_ID, cookie.getValue());
-                            long maxAgeInSecond = cookie.getMaxAge();
-                            if ( maxAgeInSecond == 0 ){
-                                pref.put(PrefManager.Key.AUTH_ASSESSMENT_SESSION_EXPIRATION, 0);
-                            } else {
-                                pref.put(PrefManager.Key.AUTH_ASSESSMENT_SESSION_EXPIRATION, currentTime + maxAgeInSecond * 1000);
```

Comment:
The `currentTime` variable initialization can be removed now since it's no longer used.